### ISSUE TITLE
`show` help pages is unhelpful

### ISSUE THREAD

**Issue by hausdorff** [ksonnet/ksonnet#143]
For example, it barely tells you how to use it:

```
Show expanded resource definitions

Usage:
  ks show [<env>|-f <file-or-dir>] [flags]

Flags:
  -V, --ext-str stringSlice           Values of external variables
      --ext-str-file stringSlice      Read external variable from a file
  -f, --file stringArray              Filename or directory that contains the configuration to apply (accepts YAML, JSON, and Jsonnet)
  -o, --format string                 Output format.  Supported values are: json, yaml (default "yaml")
  -h, --help                          help for show
  -J, --jpath stringSlice             Additional jsonnet library search path
      --resolve-images string         Change implementation of resolveImage native function. One of: noop, registry (default "noop")
      --resolve-images-error string   Action when resolveImage fails. One of ignore,warn,error (default "warn")
  -A, --tla-str stringSlice           Values of top level arguments
      --tla-str-file stringSlice      Read top level argument from a file

Global Flags:
  -v, --verbose count[=-1]   Increase verbosity. May be given multiple times.
```

**Comment by jessicayuen** [comment]
@abiogenesis-now -- do you want to take a stab at this one?

**Comment by abiogenesis-now** [comment]
looking

**Comment by jessicayuen** [comment]
@abiogenesis-now fixed this in #164
